train/train_model.py

```diff
@@ -1,4 +1,5 @@
-from solver import Solver
+from __future__ import absolute_import
+from .solver import Solver
```

Comment:
Did you test this script in Python 3.x? Because when I use python 2.7.X to run this script, an error of 
```
ValueError: Attempted relative import in non-package
```
is raised. But when I change the second line into 
```
from solver import Solver
```
everything just works fine.
I'm not so familiar with python 3.x, I'm not sure if there's a better way to fix this problem.